spyder/plugins/editor/widgets/editor.py

```diff
@@ -2426,10 +2433,31 @@ def advance_cell(self, reverse=False):
             term.setFocus()
 
     def re_run_last_cell(self):
-        text = self.get_current_editor().get_last_cell_as_executable_code()
+        text, line = self.get_current_editor().\

```

Comment:
Per PEP 8 and our convention, we're using ``()`` for line continuations for all new code. So

```python
text, line = (self.get_current_editor()
              .get_last_cell_as_executable_code())
```